src/js/actions/guides.js

```diff
@@ -114,9 +114,13 @@ define(function (require, exports) {
             topAncestors = currentDocument.layers.selectedTopAncestors,
             topAncestorIDs = collection.pluck(topAncestors, "id"),
             visibleGuides = guides.filter(function (guide) {
-                return guide.layerID === 0 || topAncestorIDs.has(guide.layerID);
+                return guide && (guide.layerID === 0 || topAncestorIDs.has(guide.layerID));
```

Comment:
I guess there was a good reason for this, but it is a little odd that the store is holding a reference to null guides.